### PR TITLE
Update hologram for Android with CMake+Android Studio

### PR DESCRIPTION
**Pull request by ggfan**
- Remove non-used android/external/nativeactivity dir
- Update Hologram to use Android Studio's CMake

### CODE REVIEW

**Comment by ggfan** [comment]
@cnorthrop @bbilodeau @alandward please take a quick look and see if I am on the wrong direction

thanks

File: demos/smoke/android/build-and-install [hunk 1]
```diff
@@ -19,7 +19,30 @@ generate_local_properties() {
...
+            echo "$cur_dir`basename \"$1\"`"
+        else
+            echo "$cur_dir/`basename \"$1\"`"
+        fi
+    fi
+    popd > /dev/null
+}
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    glslang=$(abspath ../../../external/glslang/build/install/bin/glslangValidator)
```
Comment (cnorthrop) [inline demos/smoke/android/build-and-install]: realpath is available for macOS via homebrew, so I don't think you need this.  We can add it to the upstream OSX for Android section of BUILD.md.

**Comment by ggfan** [comment]
yeah, removed smoke from this code checkin and create PR for smoke to upstream with
https://github.com/KhronosGroup/Vulkan-LoaderAndValidationLayers/pull/1706

File: demos/smoke/android/build-and-install (hunk 1 above)
Comment (ggfan) [inline demos/smoke/android/build-and-install]: Yeah, removed now

**Comment by ggfan** [comment]
Yeah, it is updated to android-23 and minSDKVersion etc to 23.  Please take a look when you get chance.  Thank you

**Comment by cnorthrop** [comment]
Thanks, Gerry.  Cherry-picked to master.

**Pull request closed without merging**